mzLib/FlashLFQ/FlashLfqEngine.cs

```diff
@@ -919,26 +985,26 @@ private void QuantifyMatchBetweenRunsPeaks(SpectraFileInfo idAcceptorFile)
             MbrScorer scorer,
             ChromatographicPeak donorPeak,
             Tolerance mbrTol,
-            RtInfo rtInfo, 
-            Normal rtScoringDistribution,
+            RtInfo rtInfo,
             int z,
             List<IsotopicEnvelope> chargeEnvelopes)
         {
-            var donorId = donorPeak.Identifications.OrderBy(p => p.PosteriorErrorProbability).First();
-            var acceptorPeak = new ChromatographicPeak(donorId, true, idAcceptorFile);
-            IsotopicEnvelope seedEnv = chargeEnvelopes.First();
+            var donorId = donorPeak.Identifications.First();
```

Comment:
Are you certain that the first will always be correct after removing the ordering operations?

Reply:
Yes. Previously, PEP was an unused field, so no ordering was actually performed